content/en/monitors/configuration/_index.md

```diff
@@ -270,6 +270,8 @@ A `Multi Alert` monitor triggers individual notifications for each entity in a m
 
 For example, when setting up a monitor to notify you if the P99 latency, aggregated by service, exceeds a certain threshold, you would receive a **separate** alert for each individual service whose P99 latency exceeded the alert threshold. This can be useful for identifying and addressing specific instances of system or application issues. It allows you to track problems on a more granular level.
 
+#### Notification grouping
```

Comment:
Thanks for this, @DavidGrashton! Because this content also pertains to multi alert, would you be cool with nesting the heading down one more level?

```suggestion
##### Notification grouping
```